profiles/netcore3.0.props

```diff
@@ -0,0 +1,14 @@
+<Project>
+  <!-- .NET Core 3 / .NET Standard 2.1 -->
+  <PropertyGroup>
+    <FakeItEasyTargetFrameworks>netstandard2.1</FakeItEasyTargetFrameworks>
+    <ValueTaskExtensionsTargetFrameworks>netstandard2.1</ValueTaskExtensionsTargetFrameworks>
+    <ExampleTargetFrameworks>netcoreapp3.0</ExampleTargetFrameworks>
+    <SpecsTargetFrameworks>netcoreapp3.0</SpecsTargetFrameworks>
+    <IntegrationTestsTargetFrameworks>netcoreapp3.0</IntegrationTestsTargetFrameworks>
+    <UnitTestsTargetFrameworks>netcoreapp3.0</UnitTestsTargetFrameworks>
+    <AnalyzerTestsTargetFrameworks>netcoreapp3.0</AnalyzerTestsTargetFrameworks>
+    <TestHelpersTargetFrameworks>netstandard2.0</TestHelpersTargetFrameworks>

```

Comment:
When I use the `FakeItEasy.user.props` file to set the build profile to netcore3.0 (which was fun), and run `build integ`, I again get

```
D:\Sandbox\FakeItEasy\tests\FakeItEasy.Tests.TestHelpers\FakeItEasy.Tests.TestHelpers.csproj :
error NU1201: Project FakeItEasy is not compatible with netstandard2.0 (.NETStandard,Version=v2.0).
Project FakeItEasy supports: netstandard2.1 (.NETStandard,Version=v2.1) [D:\Sandbox\FakeItEasy\FakeItEasy.sln]
```

Reply:
Argh... OK, I'll *really* review and test those TFMs before I push a new fix.